sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/DocumentAnalysisClientBuilder.java

```diff
@@ -164,7 +164,7 @@ public DocumentAnalysisAsyncClient buildAsyncClient() {
         // Endpoint cannot be null, which is required in request authentication
         Objects.requireNonNull(endpoint, "'Endpoint' is required and can not be null.");
         if (audience == null) {
-            audience = FormRecognizerAudience.AZURE_RESOURCE_MANAGER_PUBLIC_CLOUD;
+            audience = DocumentAnalysisAudience.AZURE_RESOURCE_MANAGER_PUBLIC_CLOUD;
```

Comment:
DocumentAnalysisAudience seems a little off to me, though I understand why you made this change. I think that this is one case where it seems better to keep it "FormRecognizer" rather than DocumentAnalysis, since it applies to the whole service. Don't you use the same values for administration?

Reply:
+1 for FormRecognizer

Reply:
I agree, I liked FormRecognizer better too. But this came back from the arch review and feedback.
@srnagar Do you think `FormRecognizerAudience` should work better?

Reply:
We use `DocumentAnalysisAudience` in .NET as well. I'm not sure how this applies to Java since it doesn't support V2 anymore, but in .NET we tried to keep all V2 and V3 models completely separate from each other in the `FormRecognizer` and `DocumentAnalysis` namespaces, including the audience type.

If we shared `FormRecognizerAudience` between V2 and V3, users would need to add the `using Azure.AI.FormRecognizer` statement in their code, and this would pollute intellisense with all V2 models.

Reply:
We don't have any other types using the prefix `FormRecognizer`. For the user, it's a bit odd to have to use `FormRecognizer`  when we have tried to rename all clients and types to `Document*`. We use the `DocumentAnalysisServiceVersion` for both analysis and admin clients - so, this would be similar to the service version type that applies to the whole service.